structural_imbalance.py

```diff
@@ -65,7 +65,7 @@ def main(sampler_type, region, show):
         params = dict(num_reads=100,
                       chain_strength=2.0,
                       )
-        sampler = EmbeddingComposite(DWaveSampler())
+        sampler = EmbeddingComposite(DWaveSampler(solver=dict(topology__type='pegasus')))
```

Comment:
Is `pegasus` required here, or the largest QPU would suffice? (future-proof)
```suggestion
        sampler = EmbeddingComposite(DWaveSampler(solver=dict(order_by='-num_qubits')))
```

Reply:
Good question, and actually I'd like to understand the issue better first.  I'm confused, because locally I am able to run the tests with the 2000Q.

Also, @hhtong mentioned that we should unpin the SDK version in this repo, which is currently fixed to 2.1.1.  If I understand correctly, upgrading the SDK to 3 would mean that `DWaveSampler()` with no arguments would default to selecting the QPU with most quibits?

Reply:
@mcfarljm, if the SDK is version 2.1.1 and the code is not specifying `qpu=True`, maybe the CI is selecting a SW solver instead of either a 2000Q or Advantage QPU. Try unpinning and running again without restricting the choice of QPU.

Reply:
Well, I missed that. That explains it. Prior to Ocean 3, `DWaveSampler()` might select a non-QPU solver. So, maybe even a software solver with 128 qubits in Chimera topology. Best to upgrade to Ocean 3+.

Reply:
And, yes, `DWaveSampler` currently [selects a QPU with max number of qubits](https://github.com/dwavesystems/dwave-system/blob/32e9065cddeb123106b43d947877043c1a2ccc78/dwave/system/samplers/dwave_sampler.py#L161).

Reply:
Thanks, that makes more sense now.  Submitted PR #35 to unpin and switch to Ocean 3.  Assuming that works, we can close this PR.